components/cover/basic/src/index.scss

```diff
@@ -0,0 +1,70 @@
+@import '~@schibstedspain/theme-basic/lib/index';
```

Comment:
Is it really needed? It just displays a warning. And if it is really allowed, is the linter setup wrong?